messaging-activemq/src/main/java/org/wildfly/extension/messaging/activemq/ClusterConnectionDefinition.java

```diff
@@ -78,9 +79,9 @@
 
     public static final SimpleAttributeDefinition ALLOW_DIRECT_CONNECTIONS_ONLY = create("allow-direct-connections-only", BOOLEAN)
             .setDefaultValue(new ModelNode(false))
-            .setAllowNull(true)
+            .setRequired(false)
             .setAllowExpression(true)
-            .setAlternatives(CommonAttributes.DISCOVERY_GROUP)
+            .setRequires(STATIC_CONNECTORS)
```

Comment:
[~jmesnil] WDYT about this and the change to the previous L118? If you configure either CONNECTOR_REFS or DISCOVERY_GROUP_NAME the effect is the same, but if you configure neither but set this it will now fail. It can help people who've made a mistake.